### ISSUE TITLE
Be able to use arearange chart type

### ISSUE THREAD

**Issue by howe1rp** [arnoutaertgeerts/python-highcharts#34]
It would be very useful to be able to use the arearange chart type rather than just area.

**Comment by arnoutaertgeerts** [comment]
@howe1rp It seems you need an extra javascript file to make this work: http://api.highcharts.com/highcharts#plotOptions.arearange. I'll see if I find time to get this done, thanks for posting!

**Comment by howe1rp** [comment]
@arnoutaertgeerts I was able to get it to work locally by modifying the index.html file to include the additional javascript file https://cdnjs.cloudflare.com/ajax/libs/highstock/2.1.7/highcharts-more.js. I also updated the highstock.js and exporting.js files to 2.1.7 in index.html and that allowed me to use arearange. I'm not sure if that's the proper way of doing it, but it worked for my purposes.

Thanks for the great tool!
